src/components/ReactTags.js

```diff
@@ -384,6 +447,21 @@ class ReactTags extends Component {
       inputId = this.props.id,
       maxLength = this.props.maxLength;
 
+    const editingTag = this.state.editing ? (
+      <div className={this.state.classNames.tagInput}>
+        <input
+          className={this.state.classNames.tagInputField}
+          type="text"
+          aria-label={placeholder}
+          onChange={this.editInputChange.bind(this)}
```

Comment:
we need to have all the functionalities which are in `handleChange` like showing suggestions. can we reuse that function here ?